### PR TITLE
Bug fixes before launch

### PR DESCRIPTION
**Pull request by whoiswillma**

### CODE REVIEW

File: Eatery/Controllers/LookAheadViewController.swift [hunk 1]
```diff
@@ -248,9 +246,9 @@ class LookAheadViewController: UIViewController, UITableViewDataSource, UITableV
             let cell = tableView.dequeueReusableCell(withIdentifier: "EateryMenuCell") as! EateryMenuTableViewCell
             
             cell.delegate = self
-//            cell.shareMenuButton.isHidden = !hasMenuIterable(eatery)
+            //            cell.shareMenuButton.isHidden = !hasMenuIterable(eatery)
```
Comment (AAAstorga) [inline Eatery/Controllers/LookAheadViewController.swift]: can you delete commented out code

File: Eatery/Controllers/LookAheadViewController.swift [hunk 2]
```diff
@@ -84,7 +82,7 @@ class LookAheadViewController: UIViewController, UITableViewDataSource, UITableV
         let dateStrings = getDateStrings(dates)
         let headerView = UIView(frame: CGRect(x: 0, y: 0, width: view.frame.size.width, height: filterSectionHeight))
 
-        filterEateriesCell = tableView.dequeueReusableCell(withIdentifier: "FilterEateriesCell") as! FilterEateriesTableViewCell
+        filterEateriesCell = (tableView.dequeueReusableCell(withIdentifier: "FilterEateriesCell") as! FilterEateriesTableViewCell)
```
Comment (AAAstorga) [inline Eatery/Controllers/LookAheadViewController.swift]: why are there parans here? they can be removed

File: Eatery/Controllers/LookAheadViewController.swift [hunk 3]
```diff
@@ -248,9 +246,9 @@ class LookAheadViewController: UIViewController, UITableViewDataSource, UITableV
             let cell = tableView.dequeueReusableCell(withIdentifier: "EateryMenuCell") as! EateryMenuTableViewCell
             
             cell.delegate = self
-//            cell.shareMenuButton.isHidden = !hasMenuIterable(eatery)
+            //            cell.shareMenuButton.isHidden = !hasMenuIterable(eatery)
             cell.shareMenuButton.isHidden = true // temporary before hotfix
-//            cell.shareIcon.isHidden = !hasMenuIterable(eatery)
+            //            cell.shareIcon.isHidden = !hasMenuIterable(eatery)
```
Comment (AAAstorga) [inline Eatery/Controllers/LookAheadViewController.swift]: delete

**Review by AAAstorga — COMMENTED** [review]
Looks good to me, just like 3 really small comments. Great work!!

**Comment by whoiswillma** [comment]
I merged in Ethan's changes for the scroll-to-top bug in the Eateries screen

**Pull request merged**